after/queries/go/injections.scm

```diff
@@ -1,137 +1,134 @@
-;; extends
+; extends
```

Comment:
what is difference between `; extends` and `;; extends`
The help says `;; extends` but I saw someone use `; extends`

```
treesitter-query-modeline-extends
;; inherits: foo,bar
;; extends
;; extends
;;
;; inherits: baz
```

Reply:
It is `; extends` not `;; extends`, the help should be updated..